website/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "terraform-plugin-mux-docs-preview",
+  "private": "true",
+  "scripts": {
+    "build": "./scripts/website-build.sh"
+  },
+  "devDependencies": {
+    "next": "^12.1.0"
+  },
```

Comment:
Will the web team be managing these dependencies or should we enable something like Dependabot and manage it ourselves?

Reply:
I think the web team is fine to manage this one dependency. @BRKalow any further thoughts/opinions here?

Reply:
Yeah, no need for dependabot for this. The web team will manage!